homeassistant/components/overkiz/climate_entities/valve_heating_temperature_interface.py

```diff
@@ -1,4 +1,5 @@
 """Support for ValveHeatingTemperatureInterface."""
+
```

Comment:
There's an added space at the top at some of your recent PRs. Intentional?

Reply:
I wonder how that got in there?